src/package/compressed.rs

```diff
@@ -244,3 +280,15 @@ impl TryFrom<Bytes> for Package {
         Package::parse(tgz)
     }
 }
+
+const MANIFEST_PREFIX: &str = r#"# THIS FILE IS AUTOMATICALLY GENERATED BY BUFFRS
```

Comment:
I would put this into the `ResolvedManifest` newtype!

Reply:
f77d2faf2d04a4765aace473cea58f2ed5a09cd0